packages/harpstrata/src/get-harp-strata/utils/get-degree-matrix/get-degree-matrix.test.ts

```diff
@@ -20,7 +20,7 @@ test('getDegreeMatrix function maps a simple 2d array of 0s to 4th degrees (6) w
 test('getDegreeMatrix maps a major diatonic halfstepmatrix in to a major diatonic degreematrix in first pozition', () => {
   const { MAJOR_DIATONIC_FIRST_POZITION } = EXAMPLE_DEGREE_MATRICES
   const actualArray = getDegreeMatrix(
-    MAJOR_DIATONIC_APPARATUS.halfstepIndexMatrix,
+    majorDiatonicApparatus.halfstepIndexMatrix,
```

Comment:
Need to double-check whether there are other places where this decapitalisation should happen.

Reply:
I've not been following a well articulated convention. I think my opinion was basically that test data in particular deserves capitalisation to demonstrate that it is explicitly stated at it's source, but the edges of how I think about it are hazy.

I don't think I'll be able to remove all imprecision in my utilisation of capitalisation but I think airbnb's definition can support me here: https://github.com/airbnb/javascript/#naming--uppercase

Reply:
Much improved